content/docs/serverless/serverless-driver.md

```diff
@@ -98,10 +98,6 @@ export default async function handler(
 
 </CodeTabs>
 
-<Admonition type="note">
-The maximum request size and response size for queries over HTTP is 10 MB. Additionally, there is a 15-second proxy timeout for SQL requests over HTTP. Long-running queries that exceed the 15-second threshold are terminated.
```

Comment:
The 10MB limit still exists for now

Reply:
Thank you. Did not mean to remove that part!